### PR TITLE
Add brew/jenv installation instructions for OSX

### PR DESCRIPTION
**Pull request by imavroukakis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +31,48 @@ Early Access build of Jigsaw JDK9 is available at [http://jdk.java.net/jigsaw/](
...
-       - In case a `.dmg` file is available, download it and follow the installation steps, you will still have to update your `JAVA_HOME` and start script as mentioned in [this guide]( https://github.com/musinoli/guides/blob/master/setup_jigsaw.md).    
+       - In case a `.dmg` file is available, download it and follow the installation steps, you will still have to update your `JAVA_HOME` and start script as mentioned in [this guide]( https://github.com/musinoli/guides/blob/master/setup_jigsaw.md).
        - update your `.bashrc` with `export PATH=$JAVA_HOME/bin:$PATH`
+
+       #### Via Brew and jenv
+       You will need the latest version of brew with cask support (1.1.13 at the time of writing). If you don't have brew installed, go [here](https://brew.sh/) first.
+       - `brew install jenv`
+       - `brew cask install java9-beta`
+
+       You will need to make sure jenv is evaluated in every terminal session. Add	`eval "$(jenv init -)"` to your shell's .rc file
```
Comment (neomatrix369) [inline README.md]: Whitespaces after `Add    ` is that intended?

File: README.md [hunk 2]
```diff
@@ -31,9 +31,48 @@ Early Access build of Jigsaw JDK9 is available at [http://jdk.java.net/jigsaw/](
...
+
+       Now run `jenv versions`. The output might differ (if you have other JDK's installed) but in general
```
Comment (neomatrix369) [inline README.md]: wordwrap after general

File: README.md [hunk 3]
```diff
@@ -31,9 +31,48 @@ Early Access build of Jigsaw JDK9 is available at [http://jdk.java.net/jigsaw/](
...
+       #### Manual way
        - Please refer to [this guide]( https://github.com/musinoli/guides/blob/master/setup_jigsaw.md) - thanks Nolita ([@musinoli](https://github.com/musinoli)) for your help
-       - In case a `.dmg` file is available, download it and follow the installation steps, you will still have to update your `JAVA_HOME` and start script as mentioned in [this guide]( https://github.com/musinoli/guides/blob/master/setup_jigsaw.md).    
+       - In case a `.dmg` file is available, download it and follow the installation steps, you will still have to update your `JAVA_HOME` and start script as mentioned in [this guide]( https://github.com/musinoli/guides/blob/master/setup_jigsaw.md).
        - update your `.bashrc` with `export PATH=$JAVA_HOME/bin:$PATH`
+
+       #### Via Brew and jenv
+       You will need the latest version of brew with cask support (1.1.13 at the time of writing). If you don't have brew installed, go [here](https://brew.sh/) first.
+       - `brew install jenv`
+       - `brew cask install java9-beta`
```
Comment (neomatrix369) [inline README.md]: is there a way a list of all `java9` or `jdk9` brew packages can be enlisted ?

**Comment by neomatrix369** [comment]
Looks really good @imavroukakis thanks for doing this, hopefully this will put the MacOSX users at ease.

File: README.md (hunk 3 above)
Comment (imavroukakis) [inline README.md]: There's only one java9 package at the moment
```
➜  test brew search java
app-engine-java                                 google-java-format                              javarepl                                        jslint4java                                     libreadline-java
Caskroom/cask/eclipse-java                                  Caskroom/cask/netbeans-java-ee                              Caskroom/versions/charles-applejava-beta                    Caskroom/versions/java6
Caskroom/cask/java-jdk-javadoc                              Caskroom/cask/netbeans-java-se                              Caskroom/versions/charles-applejava                         Caskroom/versions/java7
Caskroom/cask/java                                          Caskroom/cask/yourkit-java-profiler                         Caskroom/versions/java-beta                                 Caskroom/versions/java9-beta
```

File: README.md (hunk 1 above)
Comment (imavroukakis) [inline README.md]: err no :)

**Comment by imavroukakis** [comment]
@neomatrix369 changes done

**Comment by neomatrix369** [comment]
Looks good @imavroukakis, thanks again!

**Pull request merged**